src/command_modules/azure-cli-consumption/azure/cli/command_modules/consumption/_params.py

```diff
@@ -15,3 +15,16 @@ def load_arguments(self, _):
         c.argument('start_date', options_list=['--start-date', '-s'], type=get_datetime_type(), help='start date (in UTC Y-m-d) of the usages. Both start date and end date need to be supplied or neither')
         c.argument('end_date', options_list=['--end-date', '-e'], type=get_datetime_type(), help='end date (in UTC Y-m-d) of the usages. Both start date and end date need to be supplied or neither')
         c.argument('billing_period_name', options_list=['--billing-period-name', '-p'], help='name of a specific billing period to get the usage details that associate with')
+    
+    with self.argument_context('consumption reservations summaries list') as rs:
+        rs.argument('reservationorderid', options_list=['--reservationorderid', '-roid'],help='Reservation order id')
+        rs.argument('reservationid', options_list=['--reservationid', '-rid'], help='Reservation id')
```

Comment:
The short form (single dash) can only have a single letter.
i.e. `-roid` and `-rid`

Reply:
i am able to use multiple letters ....
-roid and -rid works ......

Reply:
Changed it to single letter